### PR TITLE
COBRA-3961 Changed IMAGE_PULL_SECRETS to be ',' separated list

### PR DESCRIPTION
**Pull request by nedhanks**

### CODE REVIEW

File: space/space.go [hunk 1]
```diff
@@ -81,19 +81,20 @@ func Createspace(db *sql.DB, space structs.Spacespec, berr binding.Errors, r ren
 		utils.ReportError(err, r)
 		return
 	}
-	if os.Getenv("IMAGE_PULL_SECRET") != "" {
-		// This secret created used to be passed in to AddImagePullSecretToSpace, but that method
-		// did nothing with it, so we don't in the refactor.
-		if err := rt.CopySecret(os.Getenv("IMAGE_PULL_SECRET"), "akkeris-system", space.Name); err != nil {
+
+	ipss := []structs.Namespec{}
```
Comment (calaway) [inline space/space.go:85]: What does `ipss` stand for here? Is that something everyone else would be familiar with? If so, great. If not, maybe consider changing the variable name to something more descriptive? Just a suggestion, so your call.

**Review by calaway — APPROVED** [review]
Approved with just one comment/suggestion.

File: space/space.go (hunk 1 above)
Comment (nedhanks) [reply space/space.go:85]: ImagePullSecretS - plural

Comment (nedhanks) [reply space/space.go:85]: I should use more descriptive names.

**Pull request merged**